### PR TITLE
XML templates for eCDF

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
New templates for financial reports with eCDF codes.

**Templates updated** : 
- Luxembourg Balance Sheet
- Luxembourg Balance Sheet (abbreviated)
- Luxembourg Profit & Loss
- Luxembourg Profit & Loss (abbreviated)

**New template** : 
- Luxembourg Chart of Accounts

### CODE REVIEW

File: l10n_lu_mis_reports/data/mis_report_pl.xml [hunk 1]
```diff
@@ -191,291 +191,291 @@
...
     </record>
-    <record model="mis.report.kpi" id="mis_report_pl_fr392">
-      <field name="report_id" ref="mis_report_pl"/>
-      <field name="name">fr392</field>
+    <record id="mis_report_pl_fr153" model="mis.report.kpi">
+      <field ref="mis_report_pl" name="report_id"/>
+      <field name="name">fr153</field>
       <field name="description">RESULTAT DE L'EXERCICE A TRANSFERER AU 142</field>
-      <field name="expression">-balp[resultat,resultat-transfert]</field>
+      <field name="expression">-balp[resultat%,resultat-transfert%]</field>
```
Comment (sbidoul) [inline l10n_lu_mis_reports/data/mis_report_pl.xml]: remove %

File: l10n_lu_mis_reports/data/mis_report_pl.xml [hunk 2]
```diff
@@ -191,291 +191,291 @@
...
     </record>
-    <record model="mis.report.kpi" id="mis_report_pl_fr465">
-      <field name="report_id" ref="mis_report_pl"/>
-      <field name="name">fr465</field>
+    <record id="mis_report_pl_ecdf_746_745" model="mis.report.kpi">
+      <field ref="mis_report_pl" name="report_id"/>
+      <field name="name">ecdf_746_745</field>
       <field name="description">9. Quote-part dans le profit des entreprises mises en équivalence</field>
-      <field name="expression">balp[757%]</field>
+      <field name="expression">-balp[757%]</field>
```
Comment (sbidoul) [inline l10n_lu_mis_reports/data/mis_report_pl.xml]: Attention: minus sign

**Comment by sbidoul** [comment]
In mis_builder_ca:
- kpi 1, 2, ..., 6, 7 must be renamed as numbers are not valid python identifiers (I'm wondering why `mis_builder` does not catch this error at import time, btw)
- kpi `Bilan` and `Resultat` are be redundant with `ecdf_1111_1112` and `ecdf_2257_2258`, so I'd remove them and move the latter so they are at the right place in the sequence
- `bale[resultat%]` should be `bal[resultat%]`

BS and PL look good. Great job!

**Comment by ThomasBinsfeld** [comment]
@sbidoul  If I remove KPI `Resultat`, the expression of the KPI `ecdf_2257_2258` left will be `bale[6,7]` and there will be no `bale[resultat%]`anymore, right ?

**Comment by sbidoul** [comment]
resultat is a view account with two childrens (6, 7) so that should be equivalent.

File: l10n_lu_mis_reports/data/mis_report_ca.xml [hunk 3]
```diff
@@ -6232,7 +6232,7 @@
     </record>
     <record id="mis_report_ca_6" model="mis.report.kpi">
       <field ref="mis_report_ca" name="report_id"/>
-      <field name="name">6</field>
+      <field name="name">Classe6</field>
       <field name="description">6  - CLASSE 6 - COMPTES DE CHARGES</field>
       <field name="expression">bale[6%]</field>
```
Comment (sbidoul) [inline l10n_lu_mis_reports/data/mis_report_ca.xml]: This must be `bal[6%]`

File: l10n_lu_mis_reports/data/mis_report_ca.xml [hunk 4]
```diff
@@ -10302,7 +10302,7 @@
     </record>
     <record id="mis_report_ca_7" model="mis.report.kpi">
       <field ref="mis_report_ca" name="report_id"/>
-      <field name="name">7</field>
+      <field name="name">Classe7</field>
       <field name="description">7  - CLASSE 7 - COMPTES DE PRODUITS</field>
       <field name="expression">bale[7%]</field>
```
Comment (sbidoul) [inline l10n_lu_mis_reports/data/mis_report_ca.xml]: s/bale/bal/

File: l10n_lu_mis_reports/data/mis_report_ca.xml [hunk 5]
```diff
@@ -6219,10 +6219,10 @@
...
       <field name="sequence">13220</field>
     </record>
-    <record id="mis_report_ca_resultat" model="mis.report.kpi">
+    <record id="mis_report_ca_ecdf_2257_2258" model="mis.report.kpi">
       <field ref="mis_report_ca" name="report_id"/>
-      <field name="name">Resultat</field>
-      <field name="description">RESULTAT</field>
+      <field name="name">ecdf_2257_2258</field>
+      <field name="description">Resultat - TOTAL CLASSES 6 ET 7</field>
       <field name="expression">bale[6,7]</field>
```
Comment (sbidoul) [inline l10n_lu_mis_reports/data/mis_report_ca.xml]: s/bale[6,7]/bal[6%,7%]/

File: l10n_lu_mis_reports/data/mis_report_ca.xml [hunk 6]
```diff
@@ -4,10 +4,10 @@
...
       <field name="name">Luxembourg Chart of Accounts</field>
     </record>
-    <record id="mis_report_ca_bilan" model="mis.report.kpi">
+    <record id="mis_report_ca_ecdf_1111_1112" model="mis.report.kpi">
       <field ref="mis_report_ca" name="report_id"/>
-      <field name="name">Bilan</field>
-      <field name="description">BILAN</field>
+      <field name="name">ecdf_1111_1112</field>
+      <field name="description">Bilan - TOTAL CLASSES 1 A 5</field>
       <field name="expression">bale[1,2,3,4,5]</field>
```
Comment (sbidoul) [inline l10n_lu_mis_reports/data/mis_report_ca.xml]: s/1,2,3,4,5/1%,2%,3%,4%,5%/

**Comment by ThomasBinsfeld** [comment]
@sbidoul Done !

**Pull request merged**